packages/design-system-docs/src/pages/components/Dialog/ThirdPartyLink.example.jsx

```diff
@@ -24,7 +24,7 @@ class Example extends React.PureComponent {
       <div id="App" style={{ minHeight: 300 }}>
         <Button
           href="#"
-          className="ds-c-button--link ds-u-font-weight--normal"
+          className="ds-c-button--ghost ds-u-font-weight--normal"
```

Comment:
There are a bunch of other instances of the old `ds-c-button--transparent` being used in the old docs package, but I think it's safe to say we don't need to update these, because the theme-switching release should coincide with the new buttons release.